tithiwa/group.py

```diff
@@ -240,6 +240,14 @@ def _wait_for_group_info_to_load(self):
                 chatinfo = self._wait_for_presence_of_an_element(SELECTORS.CHATROOM__INFO).get_attribute('innerText')
             except:
                 pass
+    def join_group(self,url):
+        code_pos = url.index('com/')
+        code_pos += 4
+        code = url[code_pos:]
+        group_url = 'https://web.whatsapp.com/accept?code='+ code
+        driver.get(group_url)
+        time.sleep(10)
+        driver.find_element_by_xpath('//*[@id="app"]/div/span[2]/div/div/div/div/div/div/div[2]/div[2]/div/div').click()
```

Comment:
Check this [suggestion](https://github.com/Tithibots/tithiwa/pull/63#discussion_r510258807)